libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFSDKAuthErrorManager.m

```diff
@@ -99,14 +99,21 @@ - (SFAuthErrorHandlerList *)populateDefaultAuthErrorHandlerList
     self.networkFailureAuthErrorHandler = [[SFAuthErrorHandler alloc]
                                            initWithName:kSFNetworkFailureAuthErrorHandler
                                            evalOptionsBlock:^BOOL(NSError *error, SFOAuthInfo *authInfo, NSDictionary *options) {
+                                               BOOL result = NO;
```

Comment:
Moved this code into the ErrorManager. The block is invoked only if needed and the error is passed on to the next handler in other cases